libcodechecker/libhandlers/analyze.py

```diff
@@ -510,6 +542,9 @@ def main(args):
     LOG.debug("args: " + str(args))
     LOG.debug("Output will be stored to: '" + args.output_path + "'")
 
+    # Generate skip list
```

Comment:
Maybe `'Process skip list'` would be a bit better comment here.

Reply:
Done.